src/utils/import.ts

```diff
@@ -0,0 +1,80 @@
+import fs from "fs"
+import path from "path"
+import {pathToFileURL} from "url"
```

Comment:
here it's not formatted, but
I'm not sure that it was needed to re-format all to another code-style, definitely imports, order of import and private\public methods